### PR TITLE
Consistent button label in `quick-repo-deletion`

### PR DESCRIPTION
**Pull request by xnuk**
Respecting: 0d186f70335f1efa2dfbe213f16dbcf7dff6e50e

## Test URLs
Check your any forked repositories.

### Before
'Delete fork' action button preceding the 'Watch', 'Star', and 'Fork' buttons.

### After
'Delete repo' action button preceding the 'Watch', 'Star', and 'Fork' buttons.

### CODE REVIEW

**Comment by fregante** [comment]
Thanks for the PR! https://github.com/sindresorhus/refined-github/commit/0d186f70335f1efa2dfbe213f16dbcf7dff6e50e was actually undone so the feature only appears on forks. I'd rather make it clear that the button deletes a fork, rather than a scary "delete repo"

File: source/features/quick-repo-deletion.tsx [hunk 1]
```diff
@@ -61,14 +61,15 @@ async function buttonTimeout(buttonContainer: HTMLDetailsElement): Promise<boole
...
+	const buttonText = button.textContent;
 	try {
 		do {
 			button.style.transform = `scale(${1.2 - ((secondsLeft - 5) / 3)})`; // Dividend is zoom speed
 			button.textContent = `Deleting repo in ${pluralize(secondsLeft, '$$ second')}. Cancel?`;
 			await delay(1000, {signal: abortController.signal}); // eslint-disable-line no-await-in-loop
 		} while (--secondsLeft);
 	} catch {
-		button.textContent = 'Delete repo';
+		button.textContent = buttonText;
```
Comment (yakov116) [inline source/features/quick-repo-deletion.tsx:72]:
```suggestion
		button.textContent = 'Delete fork';
```

File: source/features/quick-repo-deletion.tsx [hunk 2]
```diff
@@ -61,14 +61,15 @@ async function buttonTimeout(buttonContainer: HTMLDetailsElement): Promise<boole
 
 	let secondsLeft = 5;
 	const button = select('.btn', buttonContainer)!;
+	const buttonText = button.textContent;
```
Comment (yakov116) [inline source/features/quick-repo-deletion.tsx:64]:
```suggestion
```

**Review by yakov116 — CHANGES_REQUESTED** [review]
Don't see why we need a var, just change the text.

**Comment by fregante** [comment]
Thank you for the PR!

**Pull request merged**